README.md

```diff
@@ -175,49 +175,77 @@ There are several data loading implementations of popular datasets across differ
 
 ## Frequently Asked Questions (FAQ)
 
-Q: What should I do if the existing set of DataPipes does not do what I need?
+<details>
+<summary>
+What should I do if the existing set of DataPipes does not do what I need?
+</summary>
 
-A: You can
+You can
 [implement your own custom DataPipe](https://pytorch.org/data/main/tutorial.html#implementing-a-custom-datapipe). If you
 believe your use case is common enough such that the community can benefit from having your custom DataPipe added to
 this library, feel free to open a GitHub issue. We will be happy to discuss!
+</details>
 
-Q: What happens when the `Shuffler` DataPipe is used with DataLoader?
+<details>
+<summary>
+What happens when the <code>Shuffler</code> DataPipe is used with DataLoader?
```

Comment:
Do we need to use `<code>` here? I think "`" should be sufficient.

Reply:
Yes - I wasn't going to change the backticks, but if you check the preview, backticks do not render as <code> does between summary tags, unless we add a new line after <summary>.

The problem with that is that the new line is rendered and doesn't look good, so <code> was necessary to make the font monospace.

Reply:
I see. Makes sense.